Assets/Scripts/DogAI/DogAI.cs

```diff
@@ -125,15 +125,14 @@ void moveTo(Vector2 target)
 
     public void Pet()
     {
+        Dog dog = GetComponent<DogWorldSlot>().PeekDog;
```

Comment:
I know this is from before you started working on it, but feel like we can just save this `Dog` object as a local variable instead of fetching the reference every time `Pet` is called